geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionConfigImpl.java

```diff
@@ -1030,6 +1032,20 @@ public DistributionConfigImpl(Properties nonDefault, boolean ignoreGemFirePropsF
     modifiable = false;
   }
 
+  /*
+   * When ssl-enabled-components specified CLUSTER or ALL, the slower receiver is not allowed.
```

Comment:
I noticed here and in your test method names you use "slower". Change that to "slow". It is a "slow receiver" not a "slower receiver".

Reply:
fixed.